cfme/services/workloads.py

```diff
@@ -108,9 +108,9 @@ def is_displayed(self):
                 "All Templates & Images"])
 
 
-class BaseWorkloads(NavigatableMixin):
-    def __init__(self, appliance):
-        self.appliance = appliance
+class BaseWorkloads(Navigatable):
```

Comment:
**QUESTION**
Why did you change that?

Reply:
Because I was told that NavigatableMixin is deprecated and Navigatable should be used instead.

Reply:
@quarckster Should I change this back?

Reply:
You can keep it.